component/integration/aws/s3/deploy.go

```diff
@@ -94,6 +96,33 @@ func Deploy(ctx mongo.SessionContext, actionID primitive.ObjectID, source app.In
 		return err
 	}
 
+	if len(target.CloudFrontID) > 0 {
+		if err = invalidateCache(target, session); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func invalidateCache(target app.Instance, session *session.Session) error {
+	now := time.Now().Format(time.RFC3339Nano)
+	svc := cloudfront.New(session)
+	input := &cloudfront.CreateInvalidationInput{
+		DistributionId: aws.String(target.CloudFrontID),
+		InvalidationBatch: &cloudfront.InvalidationBatch{
+			Paths: &cloudfront.Paths{
+				Items:    aws.StringSlice([]string{"/*"}),
+				Quantity: aws.Int64(1),
+			},
+			CallerReference: aws.String(now),
+		},
+	}
+
+	if _, err := svc.CreateInvalidation(input); err != nil {
```

Comment:
just assign error and return it. Do not need the if statement.

Reply:
makes sense